src/Framework/AspNetCoreAnalyzers/test/Verifiers/CSharpCodeFixVerifier.cs

```diff
@@ -63,7 +63,8 @@ public static async Task VerifyCodeFixAsync(string source, DiagnosticResult[] ex
             FixedState = { Sources = { fixedSource.ReplaceLineEndings() } },
             ReferenceAssemblies = CSharpAnalyzerVerifier<TAnalyzer>.GetReferenceAssemblies(),
             NumberOfFixAllIterations = expectedIterations,
-            CodeActionEquivalenceKey = codeActionEquivalenceKey
+            CodeActionEquivalenceKey = codeActionEquivalenceKey,
+            CodeFixTestBehaviors = CodeFixTestBehaviors.SkipLocalDiagnosticCheck,
```

Comment:
⚠️ This should ideally be applied to specific tests which are impacted by the change. Tests which require this flag represent _broken product behavior_, and if the value is set for the entire test project there is no way to know what/if anything works.

Reply:
Yep, filed https://github.com/dotnet/aspnetcore/issues/49856, please continue discussion there